### ISSUE TITLE
Radians and Degrees

### ISSUE THREAD

**Issue by SterlingPeet** [arindam1993/MultiRobotEnvironment#4]
Angles are sometimes expressed as Radians, but the Simulator initial conditions and a few other locations are in Degrees.  Choosing one convention (I recommend radians) for the whole simulator and sticking to it is less confusing.

**Comment by arindam1993** [comment]
Fair point,my idea was to implement something similar to what most Game Engines/3D Software Packages(Unity, Blender) will implement, anything user facing will allow you to set up in degrees and convert internally to a Quaternion or a Transform Matrix, I took the approach of using rotation matrices internally. I don't really understand Quaternions properly enough to implement them.
I'll consult with Prof. Tucker and refactor accordingly, thanks for pointing this out though, I understand it can be confusing!
